index.bs

```diff
@@ -2788,6 +2796,70 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
 
 </div>
 
+#### The browsingContext.handleUserPrompt Command ####  {#command-browsingContext-handleUserPrompt}
+
+The <dfn export for=commands>browsingContext.handleUserPrompt</dfn>
+command allows closing an open prompt
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextHandleUserPromptCommand = {
+        method: "browsingContext.handleUserPrompt",
+        params: BrowsingContextHandleUserPromptParameters
+      }
+
+      BrowsingContextHandleUserPromptParameters = {
+        context: BrowsingContext,
+        ? accept: bool,
+        ? userText: text,
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browsingContext.handleUserPrompt">
+
+1. Let |context id| be the value of the <code>context</code> field of
+   |command parameters|.
+
+1. Let |context| be the result of [=trying=] to [=get a browsing context=]
+   with |context id|.
+
+1. Let |accept| be the value of the <code>accept</code> field of |command
+   parameters| if present, or true otherwise.
```

Comment:
Defaulting to true is a bit unusual, at least in Web IDL, but I don't see a problem with doing it here.